htmlcss/task2/index.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Document</title>
+    <link rel="stylesheet" href="header.css">
+    <link rel="stylesheet" href="info-container.css">
+    <link rel="stylesheet" href="adv-container.css">
+    <link rel="stylesheet" href="art-container.css">
+    <link rel="stylesheet" href="footer.css">
+    <link rel="stylesheet" href="fonts.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="sub-wrapper">
```

Comment:
Do you really need `.sub-wrapper`? Maybe you could use `.wrapper` and add `padding: 0 30px`?